README.md

```diff
@@ -1,5 +1,5 @@
 # In-toto Go implementation
-[![Build Status](https://travis-ci.com/in-toto/in-toto-golang.svg?branch=master)](https://travis-ci.com/in-toto/in-toto-golang) [![Coverage Status](https://coveralls.io/repos/github/in-toto/in-toto-golang/badge.svg)](https://coveralls.io/github/in-toto/in-toto-golang) [![Build status](https://ci.appveyor.com/api/projects/status/n45pmpso0t5b40vk?svg=true)](https://ci.appveyor.com/project/in-toto/in-toto-golang)
+![build](https://github.com/in-toto/in-toto-golang/workflows/build/badge.svg) [![Coverage Status](https://coveralls.io/repos/github/in-toto/in-toto-golang/badge.svg)](https://coveralls.io/github/in-toto/in-toto-golang)
```

Comment:
Can you make the first badge a link to the GitHub build page?

Reply:
@lukpueh solved, but we can only link to the build workflow overview page. A link to the latest build is not possible (Github is working on this)

Reply:
Thanks for the fix and headsup!

Reply:
I suppose we have to first activate something for this to work on the upstream repo?

Reply:
Let's merge and see. :)